<launch>
    <rosparam command="load" file="$(find porterble)/config/settings.yaml" />
    <node pkg="porterble" name="pose_estimator_node" type="pose_estimator_node.py" />
</launch>